pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -5116,6 +5131,33 @@ PJ_DECL(pj_status_t) pjsua_acc_get_config(pjsua_acc_id acc_id,
 PJ_DECL(pj_status_t) pjsua_acc_modify(pjsua_acc_id acc_id,
                                       const pjsua_acc_config *acc_cfg);
 
+/**
+ * Send response to incoming request (e.g. SIP MESSAGE) that was
+ * processed by a UAS transaction.
+ *
+ * This function can be used when the application wants to defer a
+ * response to an incoming request that was processed by 
+ * a UAS transaction.
+ *
+ * @param acc_id        The account that will send the response.
+ * @param rdata         The received request data.
+ * @param tsx           The UAS transaction that was created for the
+ *                      incoming request.
+ * @param st_code       Status code to be sent (e.g., 200 for OK, 400 for
+ *                      Bad Request, etc.).
+ * @param st_text       Optional status text. If NULL, default status text
+ *                      for the status code will be used.
+ * @param hdr_list      Optional list of headers to be added to the response.
+ *
+ * @return              PJ_SUCCESS on success, or the appropriate error code.
+ */
+PJ_DECL(pj_status_t) pjsua_acc_send_response(pjsua_acc_id acc_id,
+                                             pjsip_rx_data *rdata,
+                                             pjsip_transaction *tsx,
+                                             int st_code,
+                                             const pj_str_t *st_text,
+                                             const pjsip_hdr *hdr_list);
+
```

Comment:
I think it is better to use `const pjsua_msg_data *msg_data` instead of `hdr_list` if possible.